src/StreamJsonRpc/Reflection/MessageFormatterHelper.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace StreamJsonRpc
+{
```

Comment:
Maybe move this into the Reflection sub-namespace to remove it from the common types found in the root namespace.